pkg/command/plugin.go

```diff
@@ -124,7 +125,8 @@ func newPluginCmd() *cobra.Command {
 func newListPluginCmd() *cobra.Command {
 	var listCmd = &cobra.Command{
 		Use:   "list",
-		Short: "List available plugins",
+		Short: "List installed plugins",
+		Long:  "Display all the installed plugins",
```

Comment:
No need to use different terminology I think:
```suggestion
		Long:  "List all currently installed plugins",
```